### ISSUE TITLE
Change player marker update to ticks

### ISSUE THREAD

**Issue by kacperleague9** [jpenilla/squaremap#13]
As in title, this will let me use smaller intervals.

**Comment by Folas1337** [comment]
I agree this would be a good addition, I also get some disapproval from players for the fact that it updates a little slow.
